ui/components/ui/menu/menu-item.tsx

```diff
@@ -49,15 +61,20 @@ const MenuItem = React.forwardRef(
               name={IconName.FullCircle}
               size={IconSize.Xs}
               color={IconColor.primaryDefault}
-              style={{ '--size': '10px' }}
+              style={{ '--size': '10px' } as any}
```

Comment:
What do you think about this? It's a little involved, but I think it's worth it to avoid introducing `any` into the code.

```suggestion
              style={{ '--size': '10px' } as React.CSSProperties & { '--size': string }}
```